src/components/MediaPlayer/MediaPlayer.js

```diff
@@ -244,6 +244,70 @@ const MediaPlayer = ({ enableFileDownload = false, enablePIP = false }) => {
       ? playerConfig.sources[srcIndex]
       : playerConfig.sources,
     tracks: playerConfig.tracks,
+    userActions: {
+      hotkeys: function(event) {
+        // event.which key code values found at: https://css-tricks.com/snippets/javascript/javascript-keycodes/
+
+        // Prevent default browser actions so that page does not react when hotkeys are used.
+        // e.g. pressing space will pause/play without scrolling the page down.
+        event.preventDefault();
+
+        // Space and k toggle play/pause
+        if (event.which === 32 || event.which === 75) {
+          if (this.paused()) {
+            this.play();
+          } else {
+            this.pause();
+          }
+        }
+
+        // Adapted from https://github.com/videojs/video.js/blob/bad086dad68d3ff16dbe12e434c15e1ee7ac2875/src/js/control-bar/mute-toggle.js#L56
+        // m toggles mute
+        if (event.which === 77) {
+          const vol = this.volume();
+          const lastVolume = this.lastVolume_();
+
+          if (vol === 0) {
+            const volumeToSet = lastVolume < 0.1 ? 0.1 : lastVolume;
+
+            this.volume(volumeToSet);
+            this.muted(false);
+          } else {
+            this.muted(this.muted() ? false : true);
+          }
+        }
+
+        // f toggles fullscreen
+        // Fullscreen should only be available for videos
+        if (event.which === 70 && !this.isAudio()) {
+          if (!this.isFullscreen()) {
+            this.requestFullscreen();
+          } else {
+            this.exitFullscreen();
+          }
+        }
+
+        // Right arrow seeks 5 seconds ahead
+        if (event.which === 39) {
+          this.currentTime(this.currentTime() + 5);
+        }
+
+        // Left arrow seeks 5 seconds back
+        if (event.which === 37) {
+          this.currentTime(this.currentTime() - 5);
```

Comment:
Do we need a check here for the duration of the media in case the `currentTime` after the 5 second adjustment is either greater than the duration or less than 0 or the alternate start time of the media?
Or is it getting handled on the VideoJS end of things?

Reply:
It looks like it is getting handled on the VideoJS end of things. Trying to seek back past the start time of a video/segment gets set to the start time. Seeking past the end of a video/segment either triggers the autoplay of the next section or, if there are no more sections, triggers the "Transcripts undefined" error we've recently been seeing which is unrelated to this specific seeking behavior. 

Neither case triggers any other error messages.